src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -235,24 +235,9 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
       .length,
   )
 
-  const isRunepoolWithdrawUnlocked = useMemo(() => {
-    if (!isRunePoolUserStakingOpportunity(earnOpportunityData)) return
-
-    const maturityDate = new Date(earnOpportunityData.maturity)
-    const currentDate = new Date()
-
-    return currentDate >= maturityDate
-  }, [earnOpportunityData])
-
-  const runepoolSecondsLeftBeforeWithdrawal = useMemo(() => {
-    if (!isRunePoolUserStakingOpportunity(earnOpportunityData)) return 0
-
-    const maturityDate = new Date(earnOpportunityData.maturity)
-    const currentDate = new Date()
-
-    const diffTime = maturityDate.getTime() - currentDate.getTime()
-
-    return Math.ceil(diffTime / 1000)
+  const remainingLockupTime = useMemo(() => {
+    if (!isSaversUserStakingOpportunity(earnOpportunityData)) return 0
```

Comment:
return undefined/null instead of 0?

Reply:
Though we shouldn't really end up here so...

Reply:
going to leave this one as is. matches previous and 0 makes sense as in non savers have no lockup.